### PR TITLE
 update ak receiver transforms

### PR DESCRIPTION
**Pull request by GilmoreA6**
This PR updates the AK receiver transforms to conform to their required specs

**If you are suggesting a fix for a currently exploitable issue, please disclose the issue to the prime-reportstream team directly outside of GitHub instead of filing a PR, so we may immediately patch the affected systems before a disclosure. See [SECURITY.md/Reporting a Vulnerability](https://github.com/CDCgov/prime-reportstream/blob/master/SECURITY.md#reporting-a-vulnerability) for more information.**

Test Steps:
1. run ./prime test
2. run ./gradlew testIntegration

## Changes
- *re-ordered elements to appear in the order they appear in outbound message for readability
- *Added default value for MSH-5-1
- *Added valueset for FIPS county codes
- *Added valueset for patient ethnicity
- *Updated Integration Tests
- *Updated receiver settings in folder

### Testing
- [x ] Tested locally?
- [x ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### CODE REVIEW

File: prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform.yml [hunk 1]
```diff
@@ -31,16 +31,69 @@ elements:
...
+        Southeast Fairbanks: 2240
+        Valdez-Cordova: 2261
+        Wade Hampton: 2270
+        Wrangell-Petersburg: 2280
+        Yakutat: 2282
+        Yukon-Koyukuk: 2290
+
+  - name: ak-patient-ethnicity-identifier-code
+    value:
+      - 'Bundle.entry.resource.ofType(Patient).extension(%`rsext-ethnic-group`).value.coding.where(extension(%`rsext-code-index-name`).value = "identifier").code'
```
Comment (oslynn) [inline prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform.yml:69]: How two entries for value work?  Is it logical OR or AND?

File: prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AK/ak-phd.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      qualityFilter:
+        - "%messageId.exists()"
+        - "%patient.birthDate.exists()"
+        - "%patient.name.family.exists()"
+        - "%patient.name.given.count() > 0"
+        - "%specimen.type.exists()"
+        - "(%patient.address.line.exists() or %patient.address.postalCode.exists() or\
+      \ %patient.telecom.exists())"
+        - "((%specimen.collection.collectedPeriod.exists() or %specimen.collection.collected.exists())\
+      \ or %serviceRequest.occurrence.exists() or %observation.effective.exists())"
```
[the remark this answers is not in the document]
Comment (GilmoreA6) [reply prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AK/ak-phd.yml]: corrected

File: prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AK/ak-phd.yml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+      \ != 'hcintegrations'"
+      reverseTheQualityFilter: false
+      conditionFilter: []
+      deidentify: false
+      deidentifiedValue: ""
+      timing:
+        operation: "MERGE"
+        numberPerDay: 1440
+        initialTime: "00:00"
+        timeZone: "ALAKSA"
```
[the remark this answers is not in the document]
Comment (GilmoreA6) [reply prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AK/ak-phd.yml]: corrected

**Comment by GilmoreA6** [comment]
@victor-chaparro The AOE date time issue hasn't been resolved but I wanted to get all the other stuff out of the way so we could get all the testing except for that one issue out of the way while we still work on it.

File: prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform.yml (hunk 1 above)
Comment (GilmoreA6) [reply prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform.yml:69]: logical and. There were multiple ways that you could designate ethnicity, but it looks like one of them has been depracated. I've updated this to include only the one in use.

**Review by victor-chaparro — APPROVED** [review]
Looks good!

**Pull request merged**